src/apprt/gtk/Window.zig

```diff
@@ -284,7 +284,7 @@ pub fn init(self: *Window, app: *App) !void {
 
     // If Adwaita is enabled and is older than 1.4.0 we don't have the tab overview and so we
     // need to stick the headerbar into the content box.
-    if (!adwaita.versionAtLeast(1, 4, 0)) {
+    if (!adw_version.supportsBanner()) {
```

Comment:
Doesn't make sense given the above comment
```suggestion
    if (!adw_version.supportsTabOverview()) {
```

Reply:
Fixed.